src/styles/page-template/index.njk

```diff
@@ -126,6 +126,7 @@ Variables can be set with:
 | assetsURL     | string                                              | false                             | The base url where to find CSS/JS/images etc. e.g. `http://localhost:3030`                   |
 | theme         | string                                              | false                             | The name of the css file e.g. "census" if using a theme. Defaults to "main".                 |
 | title         | string                                              | true                              | The contents for the `<title>` element                                                       |
+| cspNonce      | string                                              | false                             | Configuration for `nonce` security                                                      |
```

Comment:
As the csp is abbreviated, it would be clearer if this was something like "The Content Security Policy (CSP) nonce value"